src/functions/getTestTypesById.ts

```diff
@@ -21,11 +21,12 @@ export const getTestTypesById: Handler = (event, context, callback) => {
     vehicleSubclass: Joi.string().only(Object.values(ForVehicleSubclass)).allow(null),
     vehicleWheels: Joi.number().min(0).max(99).allow(null),
     vehicleAxles: Joi.number().min(0).max(99).allow(null)
-  });
+  }).strict();
 
-  const queryParams = parseMissingQueryParams(event.queryStringParameters);
+  const numericParameters = ["vehicleAxles", "vehicleWheels"];
```

Comment:
suggest to use constants for the values. Instead of `"vehicleAxles"` use `VEHICLE_AXLE`. Can then be reused in unit tests or any other place.

Reply:
Done